src/code/audio_seqplayer.c

```diff
@@ -1743,14 +1743,18 @@ void AudioSeq_SequencePlayerProcessSequence(SequencePlayer* seqPlayer) {
     }
 
     seqPlayer->scriptCounter++;
-    seqPlayer->tempoAcc += seqPlayer->tempo;
-    seqPlayer->tempoAcc += (s16)seqPlayer->unk_0C;
 
-    if (seqPlayer->tempoAcc < gAudioCtx.tempoInternalToExternal) {
+    // Apply the tempo by controlling the number of updates run on the .seq script.
+    // Processing the .seq script every possible update will result in a tempo = maxTempo
+    // Processing the .seq script a fraction of the updates will result in a `tempo = fraction * maxTempo`
+    // where `fraction = (tempo + tempoChange) / maxTempo)`
+    // This algorithm using `tempoAcc` discretizes `(tempo + tempoChange) / maxTempo) * updates`
```

Comment:
Not sure if it helps, but this is where the algorithm was described:
https://discord.com/channels/688807550715560050/688851976746041391/1022257543126462535

Reply:
Well, not really, I understand what the code does, but I don't see what the comment means :sweat_smile: 
Note Simon's "process (tempo / factor) updates" means "process `(tempo / factor)` updates", not "process `(tempo / factor) * updates`" ? (with `factor = maxTempo` afaict)

Maybe another way to phrase it, if that's the idea, would be "this makes it so that on average, there are `(tempo + tempoChange) / maxTempo` ticks(todo) per maxTick(todo)" or similar?